src/components/App/index.spec.tsx

```diff
@@ -19,11 +19,11 @@ describe(__filename, () => {
     store = configureStore(),
     authToken = 'some-token',
   }: RenderParams = {}) => {
-    // TODO: Use shallowUntilTarget()
-    // https://github.com/mozilla/addons-code-manager/issues/15
-    const root = shallow(<App authToken={authToken} />, {
-      context: { store },
-    }).shallow();
+    const root = shallowUntilTarget({
+      componentInstance: <App authToken={authToken} />,
+      targetComponent: AppBase,
+      options: { shallowOptions: { context: { store } } },
```

Comment:
This is interesting (not related to your patch): we set the `store` to the `context` whereas in addons-frontend, it was injected as a prop, which caused issues with the react-redux v6.

Reply:
If this works, does that mean we aren't blocked on upgrading to react-redux v6?

Reply:
We're still blocked by Enzyme 😞